src/types/env.ts

```diff
@@ -3,10 +3,7 @@ import { StaticDecode } from "@sinclair/typebox";
 import "dotenv/config";
 import { StandardValidator } from "typebox-validators";
 
-export const envSchema = T.Object({
-  SUPABASE_URL: T.String(),
-  SUPABASE_KEY: T.String(),
-});
+export const envSchema = T.Object({});
```

Comment:
Maybe a comment to say that this corresponds to the environment variables

Reply:
which are taken from either the worker `dev.vars` or repo secrets and not `ubiquibot-config.yml` right?

Reply:
`process.env`, `dev.vars` or fed by the Worker yes.